politeiawww/api/www/v1/v1.go

```diff
@@ -667,21 +667,23 @@ type VerifyUserPaymentReply struct {
 
 // Users is used to request a list of users given a filter.
 type Users struct {
-	Username string `json:"username"` // String which should match or partially match a username
-	Email    string `json:"email"`    // String which should match or partially match an email
+	Username  string `json:"username"`  // String which should match or partially match a username
+	Email     string `json:"email"`     // String which should match or partially match an email
+	PublicKey string `json:"publickey"` // Pubkey used for Signature
+
 }
 
 // UsersReply is a reply to the Users command, replying with a list of users.
 type UsersReply struct {
-	TotalUsers   uint64         `json:"totalusers"`   // Total number of all users in the database
-	TotalMatches uint64         `json:"totalmatches"` // Total number of users that match the filters
-	Users        []AbridgedUser `json:"users"`        // List of users that match the filters
+	TotalUsers   uint64         `json:"totalusers,omitempty"`   // Total number of all users in the database
+	TotalMatches uint64         `json:"totalmatches,omitempty"` // Total number of users that match the filters
```

Comment:
This is always be present, right?  So it shouldn't have an `omitempty`.

Reply:
Correct. Removed.